cypress/integration/visual.spec.ts

```diff
@@ -62,4 +62,30 @@ describe('Visual', () => {
       });
     });
   });
+
+  it('show/collapsing panels button should work', () => {
+    cy.visit('/viz');
+
+    cy.findByTestId('panels').then(($panels) => {
+      expect(isInViewport($panels)).eq(true);
+    });
+
+    cy.findByRole('button', { name: 'Collapse panels' }).click();
```

Comment:
nit: I find regex safer in targetting elements with names.
```suggestion
    cy.findByRole('button', { name: /collapse panels/i }).click();
```